ajax/libs/SyntaxHighlighter/package.json

```diff
@@ -16,5 +16,37 @@
   "author": {
     "name": "Alex Gorbatchev",
     "url": "https://github.com/alexgorbatchev"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/syntaxhighlighter/syntaxhighlighter.git",
+    "fileMap": [
+      {
+        "basePath": "Scripts",
+        "files": [
+          "*.js"
+        ]
+      },
+      {
+        "basePath": "Styles",
+        "files": [
+          "*.css",
+          "*.png"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "js/*.js",
+          "sass/*.scss"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "**/*.js"
```

Comment:
I don't think we need all js files in `src`. Could you open an issue to confirm what files we need to host for browser? Thanks.

Reply:
This pattern can get the files after v4.0.0
But I think after this version, the web front-end file need be build by user.
As the lib's wiki show https://github.com/syntaxhighlighter/syntaxhighlighter/wiki/Building.
Maybe need to discuss with the author if it is possible to support the front-end files.

Many thanks.